code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -36,64 +36,65 @@
 	//Item currently being held.
 	var/obj/item/gripped_item = null
 
-/obj/item/gripper/medical
+/obj/item/gripper_medical
 	name = "medical gripper"
 	desc = "A grasping tool used to help patients up once surgery is complete, or to substitute for hands in surgical operations."
-	can_hold = list()
+	icon = 'icons/obj/device.dmi'
+	icon_state = "gripper"
 
-/obj/item/gripper/medical/attack_self(mob/user)
-	return
 
-/obj/item/gripper/medical/afterattack(atom/target, mob/living/user, proximity, params)
-	var/mob/living/carbon/human/H
-	if(!gripped_item && proximity && target && ishuman(target))
-		H = target
-		if(IS_HORIZONTAL(H))
-			H.AdjustSleeping(-10 SECONDS)
-			H.AdjustParalysis(-6 SECONDS)
-			H.AdjustStunned(-6 SECONDS)
-			H.AdjustWeakened(-6 SECONDS)
-			H.AdjustKnockDown(-6 SECONDS)
-			H.stand_up()
-			playsound(user.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-			user.visible_message( \
-				"<span class='notice'>[user] shakes [H] trying to wake [H.p_them()] up!</span>",\
-				"<span class='notice'>You shake [H] trying to wake [H.p_them()] up!</span>",\
-				)
-
-/obj/item/gripper/New()
-	..()
+/obj/item/gripper_medical/afterattack(atom/target, mob/living/user, proximity, params)
+	var/mob/living/carbon/human/pickup_target
```

Comment:
could you move this to after the if check and just directly assign a variable upon definition

Reply:
done, converted the verb to an action button as well